CHANGELOG.md

```diff
@@ -1,3 +1,221 @@
+## v0.5.0 - 2020-10-23
+
+We're happy to announce the release of Lokomotive v0.5.0(Eurostar).
+
+This release packs new features, bug fixes, code optimizations, platform updates, security hardening, and much more.
+
+### Changes in v0.5.0
+
+#### Kubernetes updates
+
+- Update Kubernetes version to
```

Comment:
I'm not a fan of this changelog format. It's difficult and confusing to read IMO.
```
- Update Kubernetes version to
```
This is an imperative sentence.

How about something like:
```suggestion
- Lokomotive now ships Kubernetes version `v1.19.3`
```

or
```suggestion
- Kubernetes version has been updated to `v1.19.3`
```

However, this might be something to discuss for the next releases.